src/minesweeper/utils/position/position.ts

```diff
@@ -0,0 +1,13 @@
+import { IPosition } from '@minesweeper/lib/types';
+import { FlagModel } from '@minesweeper/domain/models';
+
+export const getSamePosition = (columnMarked: number, rowMarked: number) => {
```

Comment:
i think this function can get two parameters (position1: IPosition, position2: IPosition) and return a boolean

```suggestion
export const haveSamePosition = (firstPosition: IPosition, secondPosition: IPosition) => {
  const [fistX, firstY] = firstPosition
    const secondX, secondY] = secondPosition
  return  fistX === secondX && firstY === secondY;
```